docs/tutorials/dhis2-docker.md

```diff
@@ -1,384 +1,41 @@
-```
-
-Start the Nginx service and enable it to start on boot.
-
-```
-sudo systemctl start nginx
-sudo systemctl enable nginx
-```
-
-## Install certbot (_Optional_)
-
-[Full installation instructions for `certbot`](https://certbot.eff.org/lets-encrypt/debianbuster-nginx).
-
-```
-sudo apt-get install certbot python-certbot-nginx
-```
+This way of running DHIS2 provides extra features like running multiple DHIS2 "clusters" at once, saving different configuration combinations and more.
 
-Run the tool to setup certificates for sites configured in Nginx:
-
-```
-sudo certbot --nginx
-```
-
-> :information_source: After setting up a certificate with certbot, usually I
-> have to clean up my Nginx `default` config manually.
-
-To simulate a renewal, use:
-
-```
-sudo certbot renew --dry-run
-```
-
-For the real deal run without the `--dry-run` switch. You will want to do this
-every month or so.
-
-## Install D2 CLI
-
-Can be installed using Yarn (recommended):
-
-```
-yarn global add @dhis2/cli
-```
-
-> :information_source: NPM can be used, but is [not
-> recommended](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally)
-> for many reasons. If Yarn is unacceptable, use `npx @dhis2/cli` instead.
-
-Add the global Yarn packages `.bin` directory to your `PATH` so that you can
-easily execute the commands:
-
-```
-tee -a ~/.bash_profile <<EOF
-if [ -d "\$HOME/.config/yarn/global/node_modules/.bin" ] ; then
-    PATH="\$HOME/.config/yarn/global/node_modules/.bin:\$PATH"
-fi
-EOF
-```
-
-For shells other than Bash, a different `.profile` file is needed, e.g.
-`.zsh_profile` for ZSH. If no shell specific profile file is found or used,
-then `.profile` would work in most cases.
-
-### D2 configuration file (_Optional_)
-
-```
-mkdir -p ~/.config/d2
-tee ~/.config/d2/config.js <<EOF
-module.exports = {
-    cluster: {
-        channel: 'stable',
-        clusters: {
-            dev: {
-                channel: 'dev',
-                dbVersion: 'dev',
-                dhis2Version: 'master',
-                customContext: false, // true if using nginx as a reverse proxy
-                port: 8080
-            },
-        },
-    },
-}
-EOF
-```
-
-## Launch the development cluster
-
-If you have the `~/.config/d2/config.js` file, starting is as easy as:
-
-```
-d2 cluster up dev
-```
-
-Without the configuration file, the equivalent is:
-
-```
-d2 cluster up dev \
-    --channel dev \
-    --dhis2-version master \
-    --db-version dev \
-    --port 8080
-```
-
-## System cleanup (_Optional_)
-
-```
-sudo apt-get autoremove
-```
+## DHIS2 in Docker in production
+We can't recommend using [the DHIS2 Docker images](https://github.com/dhis2/dhis2-core#pre-built-images) in production yet. At this point we don’t have enough experience and therefore can’t vouch for their stability for “mission critical” production use. We are not saying that someone experienced with running Docker in production shouldn’t use them.  
```

Comment:
This looks like the "old" sentence. Should we use the updated one that we iterated on in slack the other day?